### ISSUE TITLE
Difference in accuracy/slower convergence between single GPU and multi-GPU on same node

### ISSUE THREAD

**Issue by manjunaths** [horovod/horovod#478]
I am trying to parallelize the fastai cifar10 super-convergence example using Horovod. I started off from this file

https://github.com/sgugger/Adam-experiments/blob/master/train_cifar10.py

I have modified the file and attached the source code to this issue. It runs and converges. The issue I am facing is that the original code hits 83% after only 9 epochs and reaches 91+% accuracy after 30 epochs.

But when I run the code with Horovod after 30 epochs I reach only 82% accuracy. The log file of the run is attached.
I am using all the latest bits (pip installed everything) and with cuda 9.0, nccl 2.2.15 and openmpi 3.0.0.

[fastai-horovod-cifar10.zip](https://github.com/uber/horovod/files/2349677/fastai-horovod-cifar10.zip)

In general does Horovod require a larger number of epochs to converge than if on a single-GPU ?

**Comment by tgaddair** [comment]
Hey @manjunaths, I noticed in your code that you're not scaling the learning rate in proportion to the number of workers.  Per [this](https://github.com/uber/horovod/issues/384) thread and following this [example](https://github.com/uber/horovod/blob/master/examples/pytorch_imagenet_resnet50.py#L119), I'd suggest scaling the learning rate and seeing if that helps solve the convergence issue.
